src/Tribe/Admin/Notice/Install_Event_Tickets.php

```diff
@@ -169,7 +169,8 @@ public function should_display_notice_install(): bool {
 
 		return ! $this->is_installed()
 			&& empty( tribe_get_request_var( 'welcome-message-the-events-calendar' ) )
-			&& ! $this->is_install_plugin_page();
+			&& ! $this->is_install_plugin_page()
+			&& $this->is_tec_related_page();
```

Comment:
Minor: we should swap these, so that if `is_tec_related_page()` is `false`, we don't need to also check that it's not `is_install_plugin_page()`.

```suggestion
			&& $this->is_tec_related_page();
			&& ! $this->is_install_plugin_page()
```